### PR TITLE
Update _flags.html.erb

### PR DESCRIPTION
**Pull request by Dapper50**
Fixes 

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/reviewers` for help, in a comment below

### CODE REVIEW

**Comment by TildaDares** [comment]
Hi @Dapper50, can you add the issue number in the description above? It should be in the format `fixes #0000`. Thanks!

**Comment by Dapper50** [comment]
@TildaDares done

**Comment by TildaDares** [comment]
@Dapper50 The issue number is the number at the top of the issue you were assigned. It starts with a #

**Comment by Dapper50** [comment]
@TildaDares apologies for mix up, I have updated it.

**Comment by TildaDares** [comment]
Hi @Dapper50, I can't find the issue number.

File: app/views/spam2/_flags.html.erb [hunk 1]
```diff
@@ -1,46 +1,46 @@
 <script>
 $(document).ready(function () {
-	var table = table_main("#flag_table");
-	$("#all").click(function () { //select all button
-		select_all();
+    var table = table_main("#flag_table");
+    $("#all").click(function () { //select all button
+    select_all();
```
Comment (TildaDares) [inline app/views/spam2/_flags.html.erb:5]:
```suggestion
      select_all();
```

**Review by TildaDares — CHANGES_REQUESTED** [review]

File: app/views/spam2/_flags.html.erb [hunk 2]
```diff
@@ -1,46 +1,46 @@
...
+  });
+    var pageselect_val = localStorage.getItem('page-select') || '';
+   $("#pageselect").val(pageselect_val);
+   $("#batch-spam").bind('click', function (e) { //batch spam
+     batch_nav("batch_spam");
+   });
+   $("#batch-publish").bind('click', function (e) { //batch publish
+      batch_nav("batch_publish");
+   });
+   $("#batch-ban").bind('click', function (e) { //batch ban
```
Comment (TildaDares) [inline app/views/spam2/_flags.html.erb:15]:
```suggestion
  $("#batch-ban").bind('click', function (e) { //batch ban
```

File: app/views/spam2/_flags.html.erb [hunk 3]
```diff
@@ -1,46 +1,46 @@
...
+     batch_nav("batch_ban");
```
Comment (TildaDares) [inline app/views/spam2/_flags.html.erb:16]:
```suggestion
    batch_nav("batch_ban");
```

File: app/views/spam2/_flags.html.erb [hunk 4]
```diff
@@ -1,46 +1,46 @@
...
+   });
```
Comment (TildaDares) [inline app/views/spam2/_flags.html.erb:17]:
```suggestion
  });
```

File: app/views/spam2/_flags.html.erb [hunk 5]
```diff
@@ -1,46 +1,46 @@
...
+    $("#batch-unban").bind('click', function (e) { //batch ban
```
Comment (TildaDares) [inline app/views/spam2/_flags.html.erb:18]:
```suggestion
  $("#batch-unban").bind('click', function (e) { //batch ban
```

File: app/views/spam2/_flags.html.erb [hunk 6]
```diff
@@ -1,46 +1,46 @@
...
+     batch_nav("batch_unban");
```
Comment (TildaDares) [inline app/views/spam2/_flags.html.erb:19]:
```suggestion
    batch_nav("batch_unban");
```

File: app/views/spam2/_flags.html.erb [hunk 7]
```diff
@@ -1,46 +1,46 @@
...
+    });
```
Comment (TildaDares) [inline app/views/spam2/_flags.html.erb:20]:
```suggestion
  });
```

File: app/views/spam2/_flags.html.erb [hunk 8]
```diff
@@ -1,46 +1,46 @@
...
+    $("#delete-batch").bind('click', function (e) { // batch delete
```
Comment (TildaDares) [inline app/views/spam2/_flags.html.erb:21]:
```suggestion
  $("#delete-batch").bind('click', function (e) { // batch delete
```

File: app/views/spam2/_flags.html.erb [hunk 9]
```diff
@@ -1,46 +1,46 @@
...
+      batch_nav("batch_delete");
```
Comment (TildaDares) [inline app/views/spam2/_flags.html.erb:22]:
```suggestion
    batch_nav("batch_delete");
```

File: app/views/spam2/_flags.html.erb [hunk 10]
```diff
@@ -1,46 +1,46 @@
...
+   });
```
Comment (TildaDares) [inline app/views/spam2/_flags.html.erb:23]:
```suggestion
  });
```

File: app/views/spam2/_flags.html.erb [hunk 11]
```diff
@@ -1,46 +1,46 @@
...
+    $('#pageselect').change(function () {
```
Comment (TildaDares) [inline app/views/spam2/_flags.html.erb:24]:
```suggestion
  $('#pageselect').change(function () {
```

File: app/views/spam2/_flags.html.erb [hunk 12]
```diff
@@ -1,46 +1,46 @@
...
+      pagination("#pageselect", "/spam2/flags/filter/");
```
Comment (TildaDares) [inline app/views/spam2/_flags.html.erb:25]:
```suggestion
    pagination("#pageselect", "/spam2/flags/filter/");
```

File: app/views/spam2/_flags.html.erb [hunk 13]
```diff
@@ -1,46 +1,46 @@
...
+    });
```
Comment (TildaDares) [inline app/views/spam2/_flags.html.erb:26]:
```suggestion
  });
```

File: app/views/spam2/_flags.html.erb [hunk 14]
```diff
@@ -1,46 +1,46 @@
...
+   $('#spammed').on('click', function () { //spam filter
```
Comment (TildaDares) [inline app/views/spam2/_flags.html.erb:27]:
```suggestion
  $('#spammed').on('click', function () { //spam filter
```

File: app/views/spam2/_flags.html.erb [hunk 15]
```diff
@@ -1,46 +1,46 @@
...
+      search_table("spammed", "/spam2/flags/filter/");
```
Comment (TildaDares) [inline app/views/spam2/_flags.html.erb:28]:
```suggestion
    search_table("spammed", "/spam2/flags/filter/");
```

File: app/views/spam2/_flags.html.erb [hunk 16]
```diff
@@ -1,46 +1,46 @@
...
+    });
```
Comment (TildaDares) [inline app/views/spam2/_flags.html.erb:29]:
```suggestion
  });
```

File: app/views/spam2/_flags.html.erb [hunk 17]
```diff
@@ -1,46 +1,46 @@
...
+    $('#unmoderated').on('click', function () { // unmoderated filter
```
Comment (TildaDares) [inline app/views/spam2/_flags.html.erb:30]:
```suggestion
  $('#unmoderated').on('click', function () { // unmoderated filter
```

File: app/views/spam2/_flags.html.erb [hunk 18]
```diff
@@ -1,46 +1,46 @@
...
+      search_table("unmoderated", "/spam2/flags/filter/");
```
Comment (TildaDares) [inline app/views/spam2/_flags.html.erb:31]:
```suggestion
    search_table("unmoderated", "/spam2/flags/filter/");
```

File: app/views/spam2/_flags.html.erb [hunk 19]
```diff
@@ -1,46 +1,46 @@
...
+    });
```
Comment (TildaDares) [inline app/views/spam2/_flags.html.erb:32]:
```suggestion
  });
```

File: app/views/spam2/_flags.html.erb [hunk 20]
```diff
@@ -1,46 +1,46 @@
...
+    $('#page').on('click', function () { // page filter
```
Comment (TildaDares) [inline app/views/spam2/_flags.html.erb:33]:
```suggestion
  $('#page').on('click', function () { // page filter
```

File: app/views/spam2/_flags.html.erb [hunk 21]
```diff
@@ -1,46 +1,46 @@
...
+     search_table("page", "/spam2/flags/filter/");
```
Comment (TildaDares) [inline app/views/spam2/_flags.html.erb:34]:
```suggestion
    search_table("page", "/spam2/flags/filter/");
```

File: app/views/spam2/_flags.html.erb [hunk 22]
```diff
@@ -1,46 +1,46 @@
...
+   });
```
Comment (TildaDares) [inline app/views/spam2/_flags.html.erb:35]:
```suggestion
  });
```

File: app/views/spam2/_flags.html.erb [hunk 23]
```diff
@@ -1,46 +1,46 @@
...
+   $('#note').on('click', function () { // note filter
```
Comment (TildaDares) [inline app/views/spam2/_flags.html.erb:36]:
```suggestion
  $('#note').on('click', function () { // note filter
```

File: app/views/spam2/_flags.html.erb [hunk 24]
```diff
@@ -1,46 +1,46 @@
...
+      search_table("note", "/spam2/flags/filter/");
```
Comment (TildaDares) [inline app/views/spam2/_flags.html.erb:37]:
```suggestion
    search_table("note", "/spam2/flags/filter/");
```

File: app/views/spam2/_flags.html.erb [hunk 25]
```diff
@@ -1,46 +1,46 @@
...
+    });
```
Comment (TildaDares) [inline app/views/spam2/_flags.html.erb:38]:
```suggestion
  });
```

File: app/views/spam2/_flags.html.erb [hunk 26]
```diff
@@ -1,46 +1,46 @@
...
+   $('#reset').on('click', function () { // all filter
```
Comment (TildaDares) [inline app/views/spam2/_flags.html.erb:39]:
```suggestion
  $('#reset').on('click', function () { // all filter
```

File: app/views/spam2/_flags.html.erb [hunk 27]
```diff
@@ -1,46 +1,46 @@
...
+    search_table("all", "/spam2/flags/filter/");
+    });
```
Comment (TildaDares) [inline app/views/spam2/_flags.html.erb:41]:
```suggestion
  });
```

File: app/views/spam2/_flags.html.erb [hunk 28]
```diff
@@ -1,46 +1,46 @@
...
+   $('#unflag-node').tooltip({ boundary: 'window' });
```
Comment (TildaDares) [inline app/views/spam2/_flags.html.erb:42]:
```suggestion
  $('#unflag-node').tooltip({ boundary: 'window' });
```

File: app/views/spam2/_flags.html.erb [hunk 29]
```diff
@@ -1,46 +1,46 @@
...
   });
```
Comment (TildaDares) [inline app/views/spam2/_flags.html.erb:43]:
```suggestion
});
```

**Comment by TildaDares** [comment]
Feel free to commit all of those suggestions!!

**Comment by Dapper50** [comment]
Hi @TildaDares i have update the issue number with  PR #10966

**Comment by TildaDares** [comment]
@Dapper50 What do you need help with?

**Comment by Dapper50** [comment]
@TildaDares the changes I made I wanted to confirm if it is correct, or if I would require to redo it

**Comment by TildaDares** [comment]
@Dapper50 You don't have to redo anything. All you have to do is press those `Commit suggestion` buttons and we should be good.

**Comment by Dapper50** [comment]
@TildaDares okay i have done that

**Comment by TildaDares** [comment]
@Dapper50 Can you look for that button and click it? After clicking, you should see more `Commit suggestion` to click.

Screenshot 2022-04-13 at 19 59 02

**Comment by Dapper50** [comment]
@TildaDares apologies for the mix up i just finished doing it.

**Review by TildaDares — APPROVED** [review]
Great job @Dapper50. Thank you!!

**Comment by TildaDares** [comment]
Congratulations @Dapper50 🎉

**Comment by Dapper50** [comment]
@TildaDares thank you so much for your support

**Pull request merged**